views/index.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
I really like your efficient use of HTML, you clearly know your stuff! Great work Jaabir